### PR TITLE
Create biggest_Of_N_Numbers.cpp

### PR DESCRIPTION
**Pull request by born2bdev**
A code  for getting biggest number among N numbers in C++ language .

[Added a file biggest_Of_N_Numbers.cppi]

### CODE REVIEW

**Comment by Vaibhav1671998** [comment]
@arnavb Check this too :100:

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 1]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
+
+int main()
+{
+    int n, max, tmp;
+    cout << "Enter numbers of elements : ";
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: Enter the number of elements:

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 2]
```diff
@@ -0,0 +1,20 @@
+#include <iostream>
+using namespace std;
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: Don't use this.

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+
+int main()
+{
+    int n, max, tmp;
+    cout << "Enter numbers of elements : ";
+    cin >> n;
+    cout << "Enter numbers\n";
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: Enter the numbers:

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 4]
```diff
@@ -0,0 +1,20 @@
...
+    cin >> tmp;
+    max = tmp;
+    for (int i = 0; i < n - 1; i++) {
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: Pre-increment

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 5]
```diff
@@ -0,0 +1,20 @@
...
+        cin >> tmp;
+        if (max < tmp) {
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: if (tmp > max) is clearer

File: code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp [hunk 6]
```diff
@@ -0,0 +1,20 @@
...
+            max = tmp;
+        }
+    }
+    cout << "Maximum is " << max << "\n";
```
Comment (arnavb) [inline code/unclassified/biggest_Of_N_Numbers/biggest_Of_N_Numbers.cpp]: cout << "The maximum value of your integers is" << max << "\n";

**Comment by Vaibhav1671998** [comment]
I had made the changes @arnavb 
except this one -
using namespace std;

What's the need of removing it

**Comment by arnavb** [comment]
You may want to add a space after the word is in your final output.

**Comment by Vaibhav1671998** [comment]
Done @arnavb

**Comment by Vaibhav1671998** [comment]
@AdiChat Merge this too if u agree

**Comment by born2bdev** [comment]
@AdiChat See this plzz

**Comment by arnavb** [comment]
patience, @born2bdev, patience.

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**